File: POP - Weekly Practical Exercises/Week 1/MonthlyExpenditureB.c
/* Print monthly expenditure
Practical 1, Part 2 (b)
Nirvik K.C. */

/* Edit the program MonthlyExpenditureA to MonthlyExpenditureB to calculate the total money spent last month on food, leisure, and clothes.
Also, replaces those default values from the previous program with your own. */

#include <stdio.h>
int main()
{
    float foodExpenses = 350.00;
    float leisureExpenses = 175.00;
    float clothesExpenses = 100.00;
    float totalSpent;
    totalSpent = foodExpenses + leisureExpenses + clothesExpenses;
    printf("\nThe total expenditure this month was £%.2f.\n", totalSpent);
    return 0;
}

// Output : The total expenditure this month was £625.00.

File: POP - Weekly Practical Exercises/Week 6/practical6/MonthlyExpenditureB.py
# Monthly Expenditure Calculator
# Practical 6, Part 2 (b) Monthly Expenditure
# @Nirvik K.C.

# define float variables and assign values to them
foodExpenses = 300.0    # assign 300.0 to foodExpenses 
leisureExpenses = 100.0 # assign 100.0 to leisureExpenses 
clothesExpenses = 50.0  # assign 50.0 to clothesExpenses

# new variables for accommodation and travel should be of type integer
accomodationExpenses = 1500     # assign 1500 to accomodation
travelExpenses = 200    # assign 200 to travelExpenses

totalSpent = 0.0        # assign float variable for total expenses, initialised to 0.0

# Calculate the total expenses
totalSpent = (foodExpenses + leisureExpenses + clothesExpenses + accomodationExpenses + travelExpenses)

# Display the result
print("The total expenditure this month was: ", totalSpent)

# Output: The total expenditure this month was:  2150.0